be/src/exec/pipeline/fragment_context.h

```diff
@@ -155,6 +158,8 @@ class FragmentContext {
     Status _s_status;
 
     bool _enable_resource_group = false;
+
+    int _degree_of_parallelism = 1;
```

Comment:
Will this `dop` cause some confusion? Cause each pipeline in a fragment may have different dop?

Reply:
Different pipelines in a fragment must have the same `DOP`, except `ScanOperator`, which use `min(num_tablets, DOP)` as `DOP`.
Here just records `ExecFragmentRequest.pipeline_dop` to the related fragment.
Maybe it's better to rename to `_max_dop`?